src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "halls")
```

Comment:
```suggestion
@Table(name = "cinema_halls")
```

Reply:
Why should we use this style of naming? It can also get confusing when creating joined tables. Ex, cinema_halls_movie_sessions

Reply:
Read database naming convention